src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -42,13 +43,15 @@
      * @throws IOException if templateZip.zip does not exists in jar file.
      */
     public static void generateReposReport(List<RepoConfiguration> configs, String outputPath,
-            String generationDate) throws IOException {
+            String generationDate, Date sinceDate, Date untilDate) throws IOException {
         InputStream is = RepoSense.class.getResourceAsStream(TEMPLATE_FILE);
         FileUtil.copyTemplate(is, outputPath);
 
         cloneAndAnalyzeRepos(configs, outputPath);
 
-        FileUtil.writeJsonFile(new SummaryReportJson(configs, generationDate), getSummaryResultPath(outputPath));
+        FileUtil.writeJsonFile(new SummaryReportJson(
+                configs, generationDate, sinceDate, untilDate),
```

Comment:
This can be put together on the line above right (without passing the 120 characters limit)?